File: Parcados/src/modelos/Parqueadero.java
package modelos;

public class Parqueadero {
	public int cupos ; 
	public int precio  ;
	public String horario ; 
	public String caracteristicas ; 
	public String direccion ; 
	public String nombre ; 
	public Parqueadero( String _nombre , String _horario , String _caracteristicas , String _direccion ) {
		precio = -1 ; 
		cupos = -1 ;
		nombre = _nombre ; 
		horario = _horario ; 
		caracteristicas = _caracteristicas ; 
		direccion = _direccion ; 
	}
}

File: Proyecto/02 - Iteración 2/TableViewTest/TableViewTest/Parqueadero.m
//
//  Parqueadero.m
//  TableViewTest
//
//  Created by Camilo Barraza on 10/21/14.
//  Copyright (c) 2014 Camilo Barraza. All rights reserved.
//

#import "Parqueadero.h"

@implementation Parqueadero

-(id)initConNombre: (NSString *) nomb
           conZona: (NSString *) zona
        conHorario: (NSString *) hor
conCaracteristicas: (NSString *) car
      conDireccion: (NSString *) direccion
        conEmpresa: (NSString *) empresa
         conPrecio: (NSString *) prec
          conCupos: (NSString *) cup
        conLatitud: (NSString *) latitud
       conLongitud: (NSString *) longitud {
    
    
    self = [super init];
    
    if(self)
    {
        _nombre = nomb;
        _zona = zona ;
        _empresa = empresa ;
        _horario = hor ;
        _caracteristicas = car ;
        _direccion = direccion ;
        _precio = prec ;
        _cupos = cup ;
        _latitud = latitud ;
        _longitud = longitud ;
    }
    
    return self;
    
}



@end
